helm/charts/hpe-csi-driver/README.md

```diff
@@ -18,20 +18,22 @@ Depending on which [Container Storage Provider](https://scod.hpedev.io/container
 
 The following table lists the configurable parameters of the HPE-CSI chart and their default values.
 
-|  Parameter                |  Description                                                           |  Default     |
-|---------------------------|------------------------------------------------------------------------|--------------|
-| logLevel                  | Log level. Can be one of `info`, `debug`, `trace`, `warn` and `error`. | info         |
-| imagePullPolicy           | Image pull policy (`Always`, `IfNotPresent`, `Never`).                 | IfNotPresent |
-| disableNodeConformance    | Disable automatic installation of iSCSI/Multipath Packages.            | false        |
-| iscsi.chapUser            | Username for iSCSI CHAP authentication.                                | ""           |
-| iscsi.chapPassword        | Password for iSCSI CHAP authentication.                                | ""           |
-| registry                  | Registry to pull HPE CSI Driver container images from.                 | quay.io      |
+|  Parameter                  |  Description                                                           |  Default     |
+|-----------------------------|------------------------------------------------------------------------|--------------|
+| logLevel                    | Log level. Can be one of `info`, `debug`, `trace`, `warn` and `error`. | info         |
+| imagePullPolicy             | Image pull policy (`Always`, `IfNotPresent`, `Never`).                 | IfNotPresent |
+| disableNodeConformance      | Disable automatic installation of iSCSI/Multipath Packages.            | false        |
+| iscsi.chapUser              | Username for iSCSI CHAP authentication.                                | ""           |
+| iscsi.chapPassword          | Password for iSCSI CHAP authentication.                                | ""           |
+| registry                    | Registry to pull HPE CSI Driver container images from.                 | quay.io      |
+| primera3par.enabled         | Whether to install 3par related container                              | true         |
+| nimble.enabled              | Whether to install nimble related container                            | true         |
 
 It's recommended to create a [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver) file from the corresponding release of the chart and edit it to fit the environment the chart is being deployed to. Download and edit [a sample file](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver).
 
 These are the bare minimum required parameters for a successful deployment to an iSCSI environment if CHAP authentication is required.
 
-```
+```yaml
```

Comment:
Remove this. This README is rendered in far too many places that might not interpret MD properly.

Reply:
will fix